crawler/db.py

```diff
@@ -286,19 +284,21 @@ def create_dart_sql_server_conn(config: ModuleType) -> Optional[pyodbc.Connectio
 
     logger.debug(f"Attempting to connect to {dart_db_host} on port {dart_db_port}")
 
-    sql_server_conn = None
     try:
         sql_server_conn = pyodbc.connect(connection_string)
 
-        if sql_server_conn is not None:
+        if sql_server_conn:
```

Comment:
Are we sure this is the same thing? Just aware that recently we had that thing about dataframes and truthiness... so are we sure that sql_server_conn falsiness is the same as not None?

Reply:
Reverted until larger refactor.